### PR TITLE
Add 7-specific dmlc groups

### PR DESCRIPTION
**Pull request by jhbaarnh**
This depends on https://github.com/intel-innersource/applications.simulators.simics.simics-base/pull/4282

### CODE REVIEW

**Comment by syssimics** [comment]
Verification [#11849781](https://teamcity01-ir.devtools.intel.com/viewLog.html?buildId=11849781): fail

**Comment by syssimics** [comment]
Verification [#11850796](https://teamcity01-ir.devtools.intel.com/viewLog.html?buildId=11850796): fail

**Comment by syssimics** [comment]
Verification [#11854042](https://teamcity01-ir.devtools.intel.com/viewLog.html?buildId=11854042): fail

File: test/tests.py [hunk 1]
```diff
@@ -1727,10 +1727,14 @@ def test(self):
...
-    lib_files = list(map(os.path.basename, glob.glob(basedir)))
+    lib_files = set(map(os.path.basename, glob.glob(basedir)))
+    if int(simicsutils.internal.simics_version) > 6:
+        lib_files -= {"mil-std-1553.dml", "rapidio.dml",
+                      "rapidio-device.dml"}
+
     all_tests.append(ImportTest(
         'lib-dml-%s-api-%s' % (dmlver, latest_api_version),
-        dmlver, latest_api_version, lib_files,
+        dmlver, latest_api_version, list(lib_files),
```
Comment (mandolaerik) [inline test/tests.py:1737]: I'd prefer `s/list/sorted/`, determinism is nice

File: test/tests.py [hunk 2]
```diff
@@ -1727,10 +1727,14 @@ def test(self):
...
         rapidio_bank if dmlver == '1.2' else ''))
```
Comment (mandolaerik) [inline test/tests.py:1738]: the `rapidio_bank` thing is also tied to rapidio, consider adding a <=6 precondition

**Comment by syssimics** [comment]
Verification [#11855324](https://teamcity01-ir.devtools.intel.com/viewLog.html?buildId=11855324): pass

File: test/tests.py (hunk 2 above)
Comment (jhbaarnh) [reply test/tests.py]: Done

**Review by mandolaerik — APPROVED** [review]

**Pull request merged**